releases/Process/Stabilization Process.md

```diff
@@ -0,0 +1,85 @@
+3.  **Stabilize the O3DE project overall:** Our oricess ensures that bug fixes made for stabilization are reflected in the development branch.
+  
+## Stabilization Process:
+
+*Note: during this process, development still continues as normal in the development branch. The development branch is never locked and never requires exceptions.*  
+  
+### Precondition:  
+
+1.  (Release Manager) coordinates with the Technical Steering Committee, the Open 3D Foundation Marketing Committee, and the SIGs to determine a release date.
+
+### Phase 1 - Setup
+
+1.  (Release Deployment Engineer) Currently, 6 weeks before a planned release, a stabilization branch is created. This branch is named Stabilization/<version number>. Example; for the 22.10 release we would create a Stabilization/2210 branch.
+2.  (Release Deployment Engineer) The development branch, in its current state, is integrated to the newly created stabilization branch, preferably on the same day the stabilization branch is created.
+3.  (Release Manager) publishes a note in sig-all indicating that we have begun release stabilization. Encourage the community to test, report bugs, and fix them in the stabilization branch.
+4.  (Release Manager) Coordinates with sigs to create a schedule of stabilization → development integrations and maintainers responsible for those integrations.
+
+### Phase 2 - Stabilization
+
+1.  When fixing bugs for stabilization and/or a release, please fix these bugs in stabilization and let the stabilization integrators handle getting these fixes to development branch. “Cherry picking” of changes between development and stabilization branches should not be done because it can lead to problematic integrations.
+2.  (Stabilization Integrator) Twice a week, during the entire stabilization period, a contributor (Stabilization Integrator) will integrate from the stabilization branch to the development branch. These integrations require 2 reviewers and a maintainer to submit the integration. <link to integration process here>
+
+1.  The frequent cadence ensures that the commit rate per integration is low, making for an easier and quicker integration.
+2.  The frequent cadence makes it easier to identify and remove any stabilization changes that we do NOT want integrated into development. Better signal to noise ratio.
+3.  If a major conflict is encountered, the stall in the integration is not holding up a large chain of commits
+4.  It is easier to handle DCO issues with smaller integrations.
+5.  Integrations usually occur on Mondays and Thursdays. Never Fridays to reduce the chance the development branch is broken over a weekend.
+
+### Phase 3 - Soft Lock
+
+1.  (Release Manager) Once sig-release determines the stabilization branch is potentially releasable, the branch is “soft locked”. This means no changes are allowed into the branch unless the change receives an exception from sig release and the owning sig.
```

Comment:
What do you think if we add a reminder to the Release Manager to communicate the soft lock in the Discord Channel?

Reply:
This should be publicized for and wide. Discord (sig-all, sig-release, at a minimum), along with GitHub (Discussions), and we could work with the O3DF to see if we can send out an email via one of the email lists as well.

Reply:
updated in Jan 3 commit